demo/components/victory-tooltip-demo.js

```diff
@@ -172,4 +175,94 @@ class App extends React.Component {
   }
 }
 
+class VictoryTooltipInPortal extends VictoryTooltip {
+  static contextTypes = {
+    portalRegister: React.PropTypes.func,
+    portalDeregister: React.PropTypes.func
+  }
+
+  render() {
+    this.element = super.render();
+    return null;
+  }
+
+  componentDidUpdate() {
+    this.context.portalRegister(this, this.element);
+  }
+
+  componentWillUnmount() {
+    this.context.portalDeregister(this);
+  }
+}
+
+class SvgPortal extends React.Component {
+  componentWillMount() {
+    this.map = new Map();
```

Comment:
Hmm, so I definitely get why you want to avoid it, do you have any suggestion what to replace it with?

I think this needs some way of keying elements so that the same portal can render an arbitrary number of elements (otherwise the element sent by one child would replace the element previously sent by another child, since there'd be no way of knowing whether they belong to the same child or not). The most obvious key for this, to me at least, was the instance itself. Using the instance as a key requires (something like) a Map. Would you be ok with including it without polluting globals (using it as a library)?